cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/util/HttpUrlManager.kt

```diff
@@ -53,6 +55,16 @@ internal object HttpUrlManagerSingleton : HttpUrlManager {
             .build()
     }
 
+    override fun deleteCustomConsentToUrl(env: Env, params: CustomConsentReq): HttpUrl {
+        // https://cdn.privacy-mgmt.com/consent/tcfv2/consent/v3/custom/:propertyId?consentUUID={GDPR_UUID}
+        return HttpUrl.Builder()
+            .scheme("https")
+            .host(env.host)
```

Comment:
@carmelo-iriti if we use `env` only to provide `host` attribute of the object, wouldn't it be better to ask only for `host` in method declaration? In iOS I am not using `env` while constructing `url` at all, am I wrong?

Reply:
@carmelo-iriti `fun deleteCustomConsentToUrl(host: String, params: CustomConsentReq)`

Reply:
it makes sense  👍🏻